### DISCUSSION TITLE
Invalid captcha given

### DISCUSSION THREAD

**Discussion by ggtimtom** [bbernhard/signal-cli-rest-api#420] [Q&A]
Hello,

i try to register a telephonenumber, but even i i solve several captchas the token is not accepted.

It look like this:

curl -X POST -H "Content-Type: application/json" -d '{"captcha":"signal-hcaptcha.5fad97ac-7d06-4e44-b18a-b950b20148ff.registration.P1_eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.hKdwYXNza2V5xQQoVSmo5tRdDtR4lD1fwGPBv9E4HWzRBaIf2rGp0hZL3gdkzgRqEkb5WPesT-wiwtshTHW4Ern1Z5fb1kCdhFqdGAl65u0zwFOSu-09gCPesb6u6oPpq_KEny7v__DM-irNv1zHUetnt-42NYzrEveoFHtdUY0gZalWHACf5Z8FNisEaO8aTW7AR_gPy1ucPVKicAEA45pKE5ynXm-c2E0MqOoDnkS_Yr_f2ExCDxNFhrKYcgc8G2D_lFPVy58nd9T1eQkWJcyO46CfJjMiW3yKO01f5waeJyuzwEPA0JdxeKE5LlCrnl9ims_GZTYObaUdYJlT5pUk9jS5Yc-mGTRjmMkGMqL4YHKEFEEEy_1u8d4iIb2AZfX0CZu3YSMwfBKZDDqx7SA_BMnSnek1c5QExmR7dNAaZ472jS50x6Zvm40icRYszSmNxqnDZnFDGXBJxB_1XhoZhIEfi019DUddJgouzVwqri6JH6HeQyuEVVUeq0AFtZbTLrrJsWTDo-Fm032PXDed5JyohBZphcMqjiD1Z9mQxls3NnxOOs0D9B0Rj_wZzHSh9EZp6CsQ0bS--UbuM-7JudE27Pip-dg1uE8cFH2OWj5snqHgmynwG_nd34XvY5S9P0T6tmFDgHNQbRyc2760W-YpsKs_BJn25Sa_7zxeH0KDuAp_8InMpmuRjqifT3Fv8uJFqU_JCMyoqpsmE3IqXcO-LOwbSMy7fIUqsa8H8E6QvcVcyn0bTwABg2lnzShV_cioJF_XOUzL0zzShfxabHLJLJyAoeRkFjfEghWXFoAr9SQvhLJaAi67fedaHGnLIWKpa7mIB49ueAcoOiRXcg984VCxOxPTSSVwTiRQu3jPzIy4AeAR3rkEBuzBRdoy8R5iP6KkUOa4ntwiT9FPXMQmipVihGXhAfLdeZx6eM1KvN9qxA2whDxBGseNFOWDISVVqXTMVy-VpCFcH_B6EjsxA51g9RAdD-CKvSiiTtvZEY0Mb7DkR_YOm-jZXhwzmUb5K2oc2hhcmRfaWTOFDyEH6JwZAA.bz2UeRGeO_ijzq4CxXmp3RIFN-EtQTVcejVlkMVT6K0","use_voice": true}' 'http://192.168.1.108:8080/v1/register/my telephonenumber'

**Comment by bbernhard** [comment]
What is the response you are getting?

**Comment by Jones1066** [comment]
I am also having the same issue. I've pasted the response below after calling with the captcha token:

curl -X POST -H "Content-Type: application/json" -d '{"captcha":"signal-hcaptcha.5fad97ac-7d06-4e44-b18a-b950b20148ff.registration.P1_eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.hKdwYXNza2V5xQQqXdHwBiUSa4GhgukCQBpYuVCinBQmL43xQ8-SY98W2d4n7FH0Fqa_CnEhsbaoczohiXbD1SWZjJelBr5Q4fuWdQO3Zmw9knvbWtWL2XoBaEVZdfocSPC9DK6qwcEIyfyIAdIz6WIm5Ol9vi_Er18b9mR-z8IUbk0vBeHa9E9Nrcay6qFCpxD8sLfK9J6kF6VwXkj-yAU5j015j8PVFgcOIYYfP_5F-vPd4mg748zICkCs-ue_sJOsJbOc8YWcorn8Vh4gjl8F9ajPoC9L4cb77ybKYdS3oYPEnEwi_bgcm96EydZ2L1UrpRL9aG0YTf2Z3MwceqixsxxcLbfIxVIeGaXsTsJnQZ2cRmokO8G7FBD7A2s33UZO7hvy2cKQLrt5kO11Up9HlyHjG5TxtqDGAxhGkIvxcLQ8mUH8Z68-l13MMHb2fYevIhBJOmhbtpPAvvHATj5fwiD0e78ydQ3kFHXQQNJ2TzW5VViJFVBq85HxcXdm-BrxqOuOGelsMnbKpgmok3Mk19mXn6-s21fsVcIzXtsQuYMl8T6SZJzyeCSNZim2nM8ydhy1P3Z8AgEnNLYWfkWtKM6DJhLqLMopL2VnI8dEQleM0EVfSfq99fG-3agiOS8lhbLY6qFdYK8j3S6TOdCLySbStcYQ2VHhTfDyMWhgo61nZ2AwcwJvi-bmYNfcehCH4waZCt_r3WsuZBryKeBkCT_5kjQMrjsJ8L7ZPMaWXyfrMHEn7wv8VukZGnPy3oRdaYGbp74LeZyG_txk2Z9z3pDfxoFNVbgE1r7e06y87O3tvAB0CZa7zdhFmmYR3bSNR0M1X6WrvuXPo5VwHUaG4xvdTncx8_nCjuLhXTanwpqdp_aRjUCgbkVCQy6kIO79V5iHHX2m85Ilu3egc4Npns_Zt84vSyVec82oof0d8vsjUHHMz12TiZCm-uhVBn7zoJ-EuOxQKSDrH3_Uc2a2XCbz3xzlTKkQsATp5D-rUxp7XAzLkqbVvIqwnOkK_3MIoCE7HLsFcszPQqymhotB1EE3-PTCK5WvdUQOGKZI-BCt-ox0YyKKomSzXNO4WIQKVLkbks5m7T44G9H1T38_NIaVuuHZ9_MgqWhCNr5y5gi5pnQhrPMAUtznmDzqmZMhkve2k9Biqs2Tm8WyzFsnjIoMW4cA69ebxPYcdbj6xhaNv9_IsU612IvsfwW4zuYFX2xml-dC-v-8ag8hrd6JpcssYoS92nHfP9LmY59Rfnh9YZdPl8nIjJeJOIlLn3mygbR-f7SXwKvfDQu7vzYjvcRW9uVn3BiXiSwgSh6zojLmaKbs7hTo7FmGJi3w0MnxpsS7WWDhJE9FIKzsOmpEG5KHDTELYoEWfQsZ7bftFyzV88OqJJW-AOuhL6-Q-ZUimfWsM4aSRLC88kpPvGOQYa_L_KNleHDOZaazWahzaGFyZF9pZM4DMYNvonBkAA.nTKWymQMpVpZ77Jtr7gG6kvA327RL0dnYBxbiJPahxw", "use_voice": false}' 'http://127.0.0.1:8080/v1/register/<redacted by me>'
{"error":"Invalid captcha given."}

**Reply by Jones1066** [reply]
Thanks for any help!

**Reply by Jones1066** [reply]
I've gone back at this again with a fresh set of eyes and have been able to get through the registration / verify calls. Things are looking better!
